blog.md

```diff
@@ -0,0 +1,100 @@
+## Take a look at your repository from a new perspective with Git Machete Plugin!
+
+Git and IntelliJ-based software are currently one of the most important tools in software development.
+The version control system provided with the JetBrains's IDE products
+allows performing pretty much all everyday VCS actions directly via the IDE.
+Among the many useful benefits of the provided IDE and VCS integration,
+the full support of GitHub Pull Requests introduced in IDEA 2020.2 must be emphasized.
+
+Creating and maintaining small, focused, easy-to-review PRs is a comfortable and desired approach
+for both the authors and, especially, the reviewers.
+We created a plugin that eases the management of 2+ branches in the repository
+while also promoting creation of small PRs &mdash;
+**a git repository organizer, and a rebase/merge/push/pull automation tool, named Git Machete.**
+
+It allows for looking at the state of the repository from a previously inexperienceable perspective.
+It puts the primary focus on branches (rather than commits), their relationship to each other,
+and their relationship to the remote repository.
+Moreover, it provides both diagnostic and corrective functionalities
+for the shape of the repository and the mentioned relationships.
+It finds an exceptional application in maintaining linear Git history.
+
+
+### Git Machete Status Graph
+
+![Git Machete Tab with demo repository](gmt_dark.png "Git Machete Tab")
+
+The picture above presents the demo repository _filter-service_ with a few branches.
+It contains two **root branches**: _develop_ and _master_.
+The first of them has two **child branches**: _allow-ownership-link_
+(with another child branch - _build-chain_), and _call-ws_.
+
+Each line contains a name of a branch, an optional custom annotation, and relation to the remote branch.
+
+Without going into details, the **green edge** indicates the desired state -
```

Comment:
Pls use `&mdash;` or some Markdown equivalent (`--` or `---` IIRC) for mid-sentence dashes (see one of recent English Tips of the Day on Slack)